patches/bbp-virtualenv/virtualenv-extra.py

```diff
@@ -25,3 +32,24 @@ def after_install(options, home_dir):
     if not osp.exists(manylinux_mod):
         with open(manylinux_mod, 'w') as ostr:
             ostr.write('manylinux1_compatible = True\n')
+
+    # wrap around pip to call patch_wheels after each call
+    pip_bin = osp.join(home_dir, 'bin', 'pip')
+    os.rename(pip_bin, pip_bin + '.bin')
+
+    with open(pip_bin, 'w+') as file:
+        file.write(PIP_WRAPPER)
+    os.chmod(pip_bin, 0o755)
```

Comment:
I would only add execution permission, for instance: https://stackoverflow.com/questions/12791997/how-do-you-do-a-simple-chmod-x-from-within-python